src/pug/includes/body.pug

```diff
@@ -89,27 +81,21 @@
 				iframe(src="https://upscri.be/67d2fa?as_embed" loading="lazy" style="width:100%;max-width:800px;margin:0 auto;" height="400" frameborder="0" title="Subscribe for updates")
 
 	footer.flex
-		.item.flex#logo-footer
-			img(src="/assets/img/devswag-logo.svg", alt="DevSwag")
 		.item.flex
 			p Made with ❤︎ by #[a(href="https://swapnil.net/") Swapnil Agarwal]
 		.item.flex
-			p Designed by #[a(href="https://github.com/zac-garby") Zac Garby] and #[a(href="https://mihir.ch/") Mihir Chaturvedi]
-		.item.flex
-			p Redesign by #[a(href="http://github.com/anhek") Antonin Nhek]
+			p Designed by #[a(href="https://github.com/zac-garby") Zac Garby], #[a(href="https://mihir.ch/") Mihir Chaturvedi], and #[a(href="http://github.com/anhek") Antonin Nhek]
 		.item.flex
 			p Maintained by #[a(href="https://github.com/aslafy-z") Zadkiel] and #[a(href="https://github.com/vikaspotluri123") Vikas Potluri]
```

Comment:
```suggestion
			p Maintained by #[a(href="https://github.com/aslafy-z") Zadkiel] and #[a(href="https://vikaspotluri.me") Vikas Potluri]
```

Finally got around to making my website 😄

Reply:
Nice!